### ISSUE TITLE
Social Authentication?

### ISSUE THREAD

**Issue by jimmyjamieson** [mgenev/how-to-sane#74]
What about social auth being tied in? Possibly with torii?

It would be good to see an example of this, as auth is quite a difficult thing to work out.

**Comment by mgenev** [comment]
yes, indeed, thanks for bringing it up, I have to revisit auth for refreshing the tokens and I'll work on this too.
